config/tables.go

```diff
@@ -21,13 +21,14 @@ type (
 
 	//StructureTableCfg contains the names of the base level collections
 	StructureTableCfg struct {
-		ConnTable       string `default:"conn"`
-		DNSTable        string `default:"dns"`
-		HostTable       string `default:"host"`
-		HTTPTable       string `default:"http"`
-		OpenConnTable   string `default:"openconn"`
-		SSLTable        string `default:"ssl"`
-		UniqueConnTable string `default:"uconn"`
+		ConnTable            string `default:"conn"`
+		DNSTable             string `default:"dns"`
+		HostTable            string `default:"host"`
+		HTTPTable            string `default:"http"`
+		OpenConnTable        string `default:"openconn"`
+		SSLTable             string `default:"ssl"`
+		UniqueConnTable      string `default:"uconn"`
+		UniqueConnProxyTable string `default:"uconnproxy"`
```

Comment:
Could we use `uconnProxy` so that way we don't have to remember which of `beaconProxy` and `unconnProxy` is uppercase? It looks like most of the multiword collections added so far are camelCase. I don't care much if we go all lowercase or camelCase, but I would like to keep it consistent.

Reply:
Updated to camelCase for the collection name